server/src/compression.rs

```diff
@@ -812,6 +798,8 @@ pub mod test_util {
         let mut values: Vec<f32> = vec![];
 
         match (linear, random) {
+
```

Comment:
I don't think there should be a newline here.

Reply:
Fixed.